src/devices/wholeBodyDynamics/app/wholebodydynamics-icubgenova04-external.xml

```diff
@@ -0,0 +1,114 @@
+                                             0.0,0.0,1.0,0.0,0.0,0.0,
+                                             0.0,0.0,0.0,0.001,0.0,0.0,
+                                             0.0,0.0,0.0,0.0,0.001,0.0,
+                                             0.0,0.0,0.0,0.0,0.0,0.001)</param>
+       </group> -->
+
+
+<!--<group name="FT_TEMPERATURE_COEFFICIENTS">
+              <param name="l_leg_ft_sensor">(-0.1025  ,  0.2960  ,  1.2987  , -0.0106 ,   0.0023  ,  0.0037 ,  30.4064)</param>
+              <param name="r_leg_ft_sensor">(0.0,0.0,0.0,0.0,0.0,0.0,0.0)</param>
+             <param name="l_foot_ft_sensor">(0.0,0.0,0.0,0.0,0.0,0.0,0.0)</param>
+              <param name="r_foot_ft_sensor">(0.0,0.0,0.0,0.0,0.0,0.0,0.0)</param>
+       </group> -->
+
+
+     <!-- <group name="FT_OFFSET">
+              <param name="l_leg_ft_sensor">(26.0163  , -9.5655 , -60.4540 ,  -0.2150   -0.2607  ,  0.1249)</param>   
+              <param name="r_leg_ft_sensor">(0.0,0.0,0.0,0.0,0.0,0.0)</param>
+       </group> -->
+
+        <action phase="startup" level="15" type="attach">
+            <paramlist name="networks">
+                <!-- motorcontrol -->
+                <elem name="left_leg">left_leg_mc</elem>
+                <elem name="right_leg">right_leg_mc</elem>
+                <elem name="torso">torso_mc</elem>
+                <elem name="right_arm">right_arm_mc</elem>
+                <elem name="left_arm">left_arm_mc</elem>
+                <elem name="head">head_mc</elem>
+                <!-- virtual torque sensors -->
+                <!--<elem name="left_leg_vsens">left_leg_virtual_strain</elem>
+                <elem name="right_leg_vsens">right_leg_virtual_strain</elem>
+                <elem name="torso_vsens">torso_virtual_strain</elem>
+                <elem name="right_arm_vsens">right_arm_virtual_strain</elem>
+                <elem name="left_arm_vsens">left_arm_virtual_strain</elem>-->
+                <!-- imu -->
+                <elem name="imu">inertial</elem>
+                <!-- ft -->
+                <elem name="l_arm_ft_sensor">left_upper_arm_strain</elem>
+                <elem name="r_arm_ft_sensor">right_upper_arm_strain</elem>
+                <elem name="l_leg_ft_sensor">left_upper_leg_strain</elem>
+                <elem name="r_leg_ft_sensor">right_upper_leg_strain</elem>
+                <elem name="l_foot_ft_sensor">left_lower_leg_strain</elem>
+                <elem name="r_foot_ft_sensor">right_lower_leg_strain</elem>
+                <elem name="l_leg_ft_sensorMAS">left_upper_leg_strain2</elem>
+                <elem name="r_leg_ft_sensorMAS">right_upper_leg_strain2</elem>
+                <elem name="l_foot_ft_sensorMAS">left_lower_leg_strain2</elem>
+                <elem name="r_foot_ft_sensorMAS">right_lower_leg_strain2</elem>
+            </paramlist>
+        </action>
+
+        <action phase="shutdown" level="2" type="detach" />
+
+    </device>
+</devices>
```

Comment:
Yeah. Any suggestions? 
Since is related to the temperature and the tests on iCubGenova04, I can think of:
- In the [issue ](https://github.com/dic-iit/element_ft-nonlinear-modelling/issues/10) in which I describe the datasets used for the paper and added as files needed to replicate the experiments. 
- In element [element_ft-nonlinear-modelling](https://github.com/dic-iit/element_ft-nonlinear-modelling).
- In the [insitu-ft-analysis repo](https://github.com/robotology-playground/insitu-ft-analysis/tree/master/docs/usedInRobot), which is public, and has the code to generate the temperature compensation. Although the code inside has dependency on iDyntree it does not neccessarily has anything to do with robot configuration files. I have also in the past put some there, but I don't think it fits.

Reply:
Or directly in robots-configuration?

Reply:
I put them in my local robots-configuration for now